apps/web/src/pages/ItTrainingFundPage/ItTrainingFundPage.tsx

```diff
@@ -0,0 +1,578 @@
+                      description: "Heading for the fund management card",
+                    })}
+                  >
+                    <p>
+                      {intl.formatMessage({
+                        defaultMessage:
+                          "The management of the fund is the responsibility of the Interdepartmental Joint Consultation Committee, co-chaired by the Office of the Chief Information Officer and the PIPSC IT group.",
+                        id: "fKQsbL",
+                        description: "Description for the fund management card",
+                      })}
+                    </p>
+                  </CardFlat>
+                </div>
+              </div>
+            </div>
+            {/* Three types of learning opportunities */}
+            <div
+              data-h2-display="base(flex)"
+              data-h2-flex-direction="base(column)"
+              data-h2-gap="base(x1.5)"
+            >
+              <Heading
+                Icon={LightBulbIcon}
+                size="h2"
+                color="quaternary"
+                data-h2-margin="base(0)"
+              >
+                {intl.formatMessage({
+                  defaultMessage: "Three types of learning opportunities",
+                  id: "Zewopw",
+                  description:
+                    "Heading for section describing learning opportunities",
+                })}
+              </Heading>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "The fund offers 3 types of training programs to IT employees.",
+                  id: "4CpKXN",
+                  description:
+                    "First paragraph describing learning opportunities",
+                })}
+              </p>
+              <div
+                data-h2-display="base(grid)"
+                data-h2-grid-template-columns="base(1fr) p-tablet(repeat(2, minmax(0, 1fr))) l-tablet(repeat(3, minmax(0, 1fr)))"
+                data-h2-gap="base(x1)"
+              >
+                <CardBasic
+                  data-h2-overflow="base(hidden)"
+                  data-h2-padding="base(0)"
+                  data-h2-display="base(flex)"
+                  data-h2-flex-direction="base(column)"
+                >
+                  <div
+                    data-h2-display="base(block) base:children[>span](block)"
+                    data-h2-padding="base(x1)"
+                    data-h2-background-color="base(gray.darkest) base:dark(foreground.shade)"
+                    data-h2-color="base:all(white)"
+                  >
+                    <span data-h2-font-size="base(h6, 1)">
+                      {intl.formatMessage({
+                        defaultMessage: "1. Online self-paced learning",
+                        id: "mdIHDc",
+                        description: "Title for an online learning card",
+                      })}
+                    </span>
+                  </div>
```

Comment:
We should consolidate the Featured component designs to keep all the pages consistent. I'll make an issue for this.
Otherwise, can we replace the span with a heading and match the size of the homepages featured component.
```tsx
<Heading 
     level="h3"
     size="h2"
     data-h2-font-size="base(h6)"
/>
```
We also need to fix the heading levels and sizes across the site. I believe there's an issue for this.

Reply:
Thanks for making the issue!  I agree, this would be a good spot to consolidate.

Reply:
OK, I've grabbed the markup from FeatureBlock and added it in.
[semantic heading for cards](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11821/commits/9d0d3cc45d30f15eb2cf330e4b16b978af342123)
Hopefully we can get both this page and the directive page updated to use that component directly at some point.

Reply:
I noticed that it adds bolding to the titles which is in Figma but I missed before.